tests/bootstrap_project/test/project_test.dart

```diff
@@ -562,38 +588,53 @@ void main() async {
     });
   });
 
-  tearDownAll(() async {
-    try {
-      await Process.run(
-        'rm',
-        ['-rf', tempDirName],
-        workingDirectory: rootPath,
+  group('Given a created project', () {
```

Comment:
Suggestion: Add that you start docker to the description as well.

I'm suspecting this might be a source of failure in the future.

Reply:
Makes sense, good input!

Reply:
70eace7